main.go

```diff
@@ -2,10 +2,77 @@ package main
 
 import "fmt"
 
+const total = 100
 
 func main() {
 	fmt.Println("Welcome to the Task Management System")
 	fmt.Println("Project start date is: 2024-09-18 00:00:00")
 	fmt.Println("Task Management System")
+	defer func() {
+		fmt.Println(check(false))
+		if err := recover(); err != nil {
+			fmt.Println("Recovered from panic \n", err)
+		}
+	}()
 
+	fmt.Println(doneCount(total, 5))
+	taskListing(95)
+
+	fmt.Println(predicate(92))
+	fmt.Println(remainingTasks(15))
```

Comment:
Create a function that returns both the tasksRemaining and a boolean indicating if the project is near completion (e.g., if more than 90 tasks are completed).